src/Symfony/Component/Security/Core/Authorization/StrategyAwareAccessDecisionManager.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Core\Authorization;
+
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+use Symfony\Component\Security\Core\Authorization\Strategy\VoteStrategyInterface;
+use Symfony\Component\Security\Core\Authorization\Voter\VoterInterface;
+
+/**
+ * Access decision manager that takes decisions based on the configured voting strategy.
+ *
+ * @author Christian Flothmann <christian.flothmann@xabbuh.de>
+ */
+class StrategyAwareAccessDecisionManager implements AccessDecisionManagerInterface
```

Comment:
I'm not really sure why you introduced a new class here, it looks like the old class has equal behaviour?